pkg/apis/coordination/types.go

```diff
@@ -28,7 +28,7 @@ type Lease struct {
 	// +optional
 	metav1.ObjectMeta
 
-	// Specification of the Lease.
+	// spec contains the specification of the Lease.
 	// +optional
 	Spec LeaseSpec
 }
```

Comment:
github won't let me comment on unmodified lines, but also update the reference to the name of another field:

```diff
-This is measure against time of last observed RenewTime.
+This is measure against time of last observed renewTime.
```

(same comment applies to v1beta1/v1 types)

Reply:
@liggitt inculcated the required changes